backend/pkg/sqlconnect/sql-connector.go

```diff
@@ -34,24 +34,54 @@ type SqlDBTX interface {
 	BeginTx(context.Context, *sql.TxOptions) (*sql.Tx, error)
 }
 
+type SqlConnectorOption func(*sqlConnectorOptions)
+
+type sqlConnectorOptions struct {
+	mysqlDisableParseTime bool
+	postgresDriver        *string
+}
+
+// WithMysqlDisableParseTime disables MySQL time parsing
+func WithMysqlDisableParseTime(disable bool) SqlConnectorOption {
```

Comment:
Nit for future, might be clearer if this defualts to true to just have a function like:

```
func WithMysqlParseTimeDisabled() SqlConnectorOption {
  return func(opts *sqlConnectorOptions) {
    opts.mysqlDisableParseTime = true
  }
}
```